.github/workflows/cheat-client-hashes.yml

```
@@ -0,0 +1,27 @@
+name: cheat-client-hashes
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+      branches:
+        - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Generate hash list
+      run: ./generate-hash-list.sh
+
+    - name: Commit report
+      run: |
+        if [[ `git status --porcelain` ]]; then
+          git config user.name github-actions[bot]
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          git add hashes
+          git commit -m "[CI] Update cheat client hash list"
+          git push
+        fi
```

```suggestion
    - name: Create Pull Request
      uses: peter-evans/create-pull-request@v6
      with:
        commit-message: '[CI] Update cheat client hash list'
        committer: github-actions[bot]@users.noreply.github.com
        author: github-actions[bot]@users.noreply.github.com
        branch: cheat-client-hashes
        delete-branch: true
        title: '[CI] Update cheat client hash list'
        body: Update client hash lists
```
Maybe to keep track of things create PR instead of pushing directly. Later auto merge if other checks pass.

---

Also allows testing and validating this thing without making direct pushes to master branch.

---

I'm neutral to this